crayner/authenticate-bundle/1.0/config/packages/crayner_authentication.yaml

```diff
@@ -0,0 +1,46 @@
+crayner_authenticate:
+    user_class: Crayner\Authenticate\Entity\User
+    mailer_available: true
+    highest_available_encoder:
+        # Argon2i Options
+        memory_cost: 16384
```

Comment:
It seams like you added all the default config in the recipe. You should remove this and make a reference to your docs. 

You should only keep the things that are most likely to be changed by the user. Like the `rotate_password.change_every`.

Reply:
All removed.  I do see all of the settings like 'rotate_password.change_every', but understand why you would not see them that way.  Documentation already refers to all of these settings.  Thanks.